homeassistant/components/debugpy/__init__.py

```diff
@@ -43,7 +43,9 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     async def debug_start(
         call: ServiceCall | None = None, *, wait: bool = True
     ) -> None:
-        """Start the debugger."""
+        """Enable asyncio debugging and start the debugger."""
+        get_event_loop().set_debug(True)
```

Comment:
`asyncio.get_running_loop` is preferred over `get_event_loop` in later Python versions.

https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.get_event_loop